src/main/java/com/kobylynskyi/graphql/codegen/model/request/GraphQLRequestSerializer.java

```diff
@@ -37,8 +36,8 @@ public static String serialize(GraphQLRequest graphQLRequest) {
             builder.append(graphQLRequest.getResponseProjection().toString());
         }
         builder.append(" }");
-        builder.append("\"}");
-        return builder.toString();
+        String escapedString = builder.toString().replaceAll("\"", "\\\\\"");
```

Comment:
This is probably insufficient, what about all other string escapes? If the query contains input strings with unicode character escapes etc. I guess a starting point would be to escape not only quotes but also backslashes.

Second point, why the triple escape? We're escaping things that are part of the raw query (like `"`) so that it can be inserted into the JSON, so `\"`. That is one level. Now to write this escape as a Java literal, so we need to escape this backslash and quote, so `\\\"`. Why the third level?

Reply:
Could you please give an example of such query?
Because based on the queries that I tested - everything worked as expected

Reply:
Could you please give an example of such query?
Because based on the queries that I tested - everything worked as expected

Reply:
Sure, let's assume the following schema:
```
type Query {
    autocomplete(inputStr: String!): [String]!
}
```
We can write this query:
```
query AutoComplete {
    autocomplete(inputStr: "\u1234\"\n")
}
```
This means that the actual JSON to send is the following:
```
{
    "query": "query AutoComplete {\n    autocomplete(inputStr: \"\\u1234\\\"\\n\")\n}"
}
```

Reply:
Ok. Thank you so much. 
Will test and let you know.

Reply:
@joffrey-bion you were right. special characters and such things like `\n`, '\t' were a problem.
And looks like `StringEscapeUtils.escapeJava();` does the trick.